### ISSUE TITLE
Support for simplified estimation approaches

### ISSUE THREAD

**Issue by andreasnoack** [PumasAI/Pumas.jl#613]
The current (ML) estimation approach is sometimes called the 1-stage approach. It would be good also to have support for

- [ ] **Naive averaging**: Estimate a "single subject" model for a subject where the dosing and concentration data has been averaged for a population. Should support splitting the sample based on categories specified with e.g. a `by` keyword. Should output as many sets of estimates as the product of categories.
- [ ] **Naive pooled**: Estimation based on full population but without random effects. Should support splitting the sample based on categories similarly to *Naive averaging*.
- [ ] **1-stage** Estimate each subject separately. Then estimate the PK(?) relations based on the predictied values from the individual fits.

We should probably supply each of these as a dispatch similarly to how we now dispatch og the approximation of the marginal likelihood.

**Comment by andreasnoack** [comment]
Depends on #510.

**Comment by andreasnoack** [comment]
@pkofod Isn't this mostly complete?

**Comment by pkofod** [comment]
Yes, what we need is just an improved workflow. The actual fitting should work.
